### PR TITLE
Add reference to content_sets.yml schema

### PR DESCRIPTION
**Pull request by chmeliik**
* OSBS-5282

Signed-off-by: Adam Cmiel <acmiel@redhat.com>

### CODE REVIEW

File: docs/users.rst [hunk 1]
```diff
@@ -661,7 +661,10 @@ Content Sets
 The file ``content_sets.yml`` is used to define the content sets relevant to the
 container image.  This is relevant if RPM packages in container image are in
 pulp repositories. See ``pulp_repos`` in :ref:`container.yaml-compose` for how
-this file is used during build time.
+this file is used during build time. If this file is present, it must adhere
+to the `content_sets.yml schema`_.
+
+.. _`content_sets.yml schema`: https://github.com/containerbuildsystem/atomic-reactor/blob/master/atomic_reactor/schemas/content_sets.json
```
Comment (ben-alkov) [inline docs/users.rst]: URL 404s...

Comment (chmeliik) [reply docs/users.rst]: That file will be introduced by the atomic-reactor PR, I can merge it and see if it works

Comment (chmeliik) [reply docs/users.rst]: Try now

Comment (ben-alkov) [reply docs/users.rst]: :headdesk:

**Pull request merged**